src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -269,13 +268,19 @@ class SeekBar extends Slider {
    *
    * @param {EventTarget~Event} event
    *        The `mousemove` event that caused this to run.
+   * @param {boolean} mouseDown this is a flag that should be set to true if `handleMouseMove` is called directly. It allows us to skip things that should not happen if coming from mouse down but should happen on regular mouse move handler.
    *
    * @listens mousemove
    */
-  handleMouseMove(event) {
+  handleMouseMove(event, mouseDown) {
     if (!Dom.isSingleLeftClick(event)) {
       return;
     }
+
+    if (!mouseDown && !this.player_.scrubbing()) {
+      this.player_.scrubbing(true);
```

Comment:
It would be worth adding a test for this behavior.

Reply:
At first, I thought it would be too complicated to do, which it wasn't, but it was a bit tricker to get a good working test for this since we listen to mousemove on the document.

Reply:
tests now passing.